index.html

```diff
@@ -0,0 +1,63 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Portfolio Hana Sabih</title>
+    <link rel="stylesheet" href="style.css" />
+    <script src="https://code.iconify.design/iconify-icon/1.0.2/iconify-icon.min.js"></script>
+    <!-- <link href="https://fonts.googleapis.com/css?family=Material+Icons|Material+Icons+Outlined|Material+Icons+Two+Tone|Material+Icons+Round|Material+Icons+Sharp" rel="stylesheet"> -->
+  </head>
+  <body>
+    <header>
+      <h5>Welcome</h5>
+      <iconify-icon
+        icon="ic:round-menu"
+        width="24"
+        height="24"
+        class="menubar"
+      ></iconify-icon>
```

Comment:
- [ ] To follow best semantics practices, you should wrap the hamburger button in the `nav` tag to make your code look more professional.